dap-launch.el

```diff
@@ -41,7 +41,7 @@ supported."
                 (or (: "//" (* nonl) eol)
                     (: "/*" (* (or (not (any ?*))
                                    (: (+ ?*) (not (any ?/))))) (+ ?*) ?/)
-                    (: "," (group (* (any blank ?\C-j)) (any ?\} ?\])))))
+                    (: "," (group (* (any space ?\C-j )) (any ?\} ?\] )))))
```

Comment:
```suggestion
                    (: "," (group (* (any space ?\C-j)) (any ?\} ?\])))))
```
why do you insert spaces after parens?

Reply:
Wow. I thought you had requested this in the last PR, but now that I checked the history, it was totally the reverse of that. Dunno what's wrong with me :P